res/skins/QMLDemo/Mixxx/MathUtils.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable no-unused-vars */
+const clamp = function(value, min, max) {
```

Comment:
I'd prefer using ES6 Modules:
```suggestion
export function clamp(value, min, max) {
```
and rename the file `.mjs` (not sure if thats mandatory though)

Reply:
I tried this at first, but apparently Qt requires the `*.js` extension and doesn't work when the extension is `*.mjs`. When I tried this in a `*.js` file, it gives me "unexpected token export"

Reply:
https://bugreports.qt.io/browse/QTBUG-20857

Reply:
That bug has been fixed in Qt 5.0. Are you sure you are importing the file correctly? https://doc.qt.io/qt-5/qtqml-javascript-imports.html

Reply:
Sorry, I mean't to link the top comment: https://bugreports.qt.io/browse/QTBUG-20857?focusedCommentId=171486&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-171486

It doesn't mention `*.mjs`, and it only works if the file extension is `*.js`, not `*.mjs`.

Reply:
This works for me:

file "test.mjs":
```js
export function foo() {
    return "bar";
}
```
file "main.qml":
```qml
import QtQuick 2.15

import "test.mjs" as Test

Rectangle {
    width: 200; height: 200
    Text {
        text: Test.foo()
    }
}
```

Reply:
Yes. But does it work if you import it from a qmldir?

Reply:
I'll try some stuff out and issue a PR (along with the clamping issue) when I made some progress.

Reply:
Holzhaus/mixxx#9 works for me